draft-ietf-openpgp-pqc.md

```diff
@@ -755,16 +755,17 @@ Algorithm ID reference | ML-DSA    | Public key | Secret key | Signature value
 ### Signature data digest {#mldsa-sig-data-digest}
 
 Signature data (i.e. the data to be signed) is digested prior to signing operations, see [[RFC9580, Section 5.2.4]](https://www.rfc-editor.org/rfc/rfc9580#section-5.2.4).
-Composite ML-DSA + EdDSA signatures MUST use the associated hash algorithm as specified in {{tab-mldsa-hash}} for the signature data digest.
-Signatures using other hash algorithms MUST be considered invalid.
+Composite ML-DSA + EdDSA signatures MUST use an associated hash algorithm as specified in {{tab-mldsa-hash}} for the signature data digest. They SHOULD use SHA3 to match the ML-DSA internal use of SHAKE256. They MAY use SHA2 for CNSA 2.0 compliance. Signatures using other combinations or hash algorithms MUST be considered invalid.
```

Comment:
```suggestion
Composite ML-DSA + EdDSA signatures MUST use an associated hash algorithm as specified in {{tab-mldsa-hash}} for the signature data digest. They SHOULD use SHA3 to match the ML-DSA internal use of SHAKE256. They MAY use SHA2, this can for instance be necessary to satisfy specific regulatory requirements. Signatures using other combinations or hash algorithms MUST be considered invalid.
```